lib/perl/Genome/Model/Tools/Manta/Base.pm

```diff
@@ -0,0 +1,246 @@
+package Genome::Model::Tools::Manta::Base;
+
+use strict;
+use warnings;
+
+use Genome;
+use Carp qw(confess);
+
+class Genome::Model::Tools::Manta::Base {
+    is => 'Command',
+    is_abstract => 1,
+    attributes_have => [
+        tool_arg_name => {
+            is => 'String',
+            is_optional => 1,
+        },
+        tool_input_file => {
+            is => 'Boolean',
+            is_optional => 1,
+        },
+    ],
+    has_param => [
+        version => {
+            is => 'Version',
+            doc => 'The version of Manta to use.',
+            valid_values => ['0.29.6'],
+        },
+    ],
+    has_optional_output => [
+        output_files => {
+            is_many => 1,
+        },
+        input_files => {
+            is_many => 1,
+        },
+    ],
+};
+
+sub help_detail {
+    "Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads."
+}
+
+my %TOOL_VERSIONS = (
+    '0.29.6'     => '/gscmnt/gc13001/info/model_data/jwalker_scratch/src/manta-0.29.6.centos5_x86_64/bin',
```

Comment:
Sure!  It could be in configuration.  One nice thing about the packaging is we could slightly better guarantee the correct version is being used.  This way we're trusting the user to get it right.